lib/node/fullnode.js

```diff
@@ -147,6 +147,27 @@ class FullNode extends Node {
       cors: this.config.bool('cors')
     });
 
+    // Indexers
+    this.txindex = null;
+    if (this.config.bool('index-tx'))
+      this.txindex = new TXIndexer({
+        network: this.network,
+        logger: this.logger,
+        chain: this.chain,
+        memory: this.config.bool('memory'),
+        prefix: this.config.filter('index').str('prefix') || this.config.prefix
+      });
+
+    this.addrindex = null;
+    if (this.config.bool('index-address'))
```

Comment:
linting: what's common in this case? w/ braces for multiline but one statement?

Reply:
I think it's fine, plus `eslint` doesn't seem to complain.